codegen/CodeGen/Struct.pm

```diff
@@ -87,6 +101,12 @@ sub build_struct_line {
         my $output = yaml_bitfield_to_cpp_definition (preprocess_bitfield $mem);
         # Indent and append.
         $string .= indent(text => $output, indents => 1);
+    } elsif ($mem->{type} eq "struct") {
+        $mem->{instance_name} = $mem->{name};
+        delete $mem->{name};
```

Comment:
You can actually combine these lines. In Perl, `delete` returns the value it's deleting. `$mem->{instance_name} = delete $mem->{name};`

Reply:
Nice!